README.md

```diff
@@ -587,6 +587,77 @@ Once you opt-in to use additional OCI hooks, any new orchestrated containers wil
 
 * `settings.oci-hooks.log4j-hotpatch-enabled`: Enables the [hotdog OCI hooks](https://github.com/bottlerocket-os/hotdog), which are used to inject the [Log4j Hot Patch](https://github.com/corretto/hotpatch-for-apache-log4j2) into containers. Defaults to `false`.
 
+#### OCI Defaults settings
+
+Bottlerocket allows you to customize certain parts of the default [OCI spec](https://github.com/opencontainers/runtime-spec/blob/main/config.md) that is applied to workload containers.
+
+The following settings are available:
+
+##### `settings.oci-defaults.capabilities`
+
+All of the `capabilities` settings below are boolean values (`true`/`false`).
+
+The _default_ set of capabilities can be found in the following file: [sources/models/shared-defaults/oci-capabilities.toml](sources/models/shared-defaults/oci-capabilities.toml)
```

Comment:
dont send user to a source file - explicitly list off defaults as a group separately from remaining configurable settings

Reply:
also make default word bold instead of italic